ql/src/test/queries/clientnegative/prepare_execute_1.q

```diff
@@ -1,3 +0,0 @@
---! qt:dataset:src
-prepare query1 from select count(*) from src where key > ? and value < ? group by ?;
-execute query1 using 1,100,1;
```

Comment:
This query no longer fails. I have opened a follow-up to fix this https://issues.apache.org/jira/browse/HIVE-24164